app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -29,6 +29,7 @@ def index
       add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
       add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
       @admin_sets = CurationConcerns::AdminSetService.new(self).search_results(:read)
+      @presenter = presenter_class.new(SolrDocument, current_ability)
```

Comment:
The presenter class should take an instance of SolrDocument (the elements in `@admin_sets`), not the class as it's first argument.